### DISCUSSION TITLE
Would it be possible to import the badges Mozilla Ideas to GitHub?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31101] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi everyone!

As you can see in the image I won 17 badges of different levels, I wanted to import these 17 badges on GitHub. These 17 badges I've earned are from Mozilla Ideas. Would it be possible to import the badges to GitHub?

![image](https://user-images.githubusercontent.com/111701513/187063365-469534c7-bcd6-4301-b33c-4e56d9c09d7a.png)

**Comment by melvincwng** [comment, marked as answer]
@codehangen, I don't think that's possible. 

Mozilla and GitHub are two separate entities, owned by different organizations.

Hope this clarifies & answers your question.

**Reply by ghost** [reply]
@melvincwng Hi! Thank you for feedback!

**Reply by melvincwng** [reply]
@codehangen pleasure in helping :)

**Reply by ghost** [reply]
melvincwng, you were quick to respond, thank you very much. I'm glad!
